decentraland/unity-explorer
# chore: long roads

## What does this PR change?

Adds long road scenes. Ignore custom ones (around plazas and pride roads)

## How to test the changes?

Adds the long roads parcels

1. Launch the explorer
2. Go to the road coordinates mentioned above and check that the roads are perfectly placed

  - RoadCoordinate: {x: 0, y: 12}
  - RoadCoordinate: {x: 0, y: -69}
  - RoadCoordinate: {x: -70, y: -1}
  - RoadCoordinate: {x: -51, y: -49}
  - RoadCoordinate: {x: -49, y: 50}
  - RoadCoordinate: {x: -49, y: -51}
  - RoadCoordinate: {x: 2, y: 50}
  - RoadCoordinate: {x: 2, y: -51}
  - RoadCoordinate: {x: 50, y: 2}
  - RoadCoordinate: {x: 50, y: -49}
  - RoadCoordinate: {x: 13, y: 0}

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
